Core/Core/Grades/ViewModel/GradeListViewModel.swift

```diff
@@ -202,6 +168,19 @@ public final class GradeListViewModel: ObservableObject {
         selectedGroupByOption.accept(selectedSortByOption)
     }
 
+   private func getAssignmentsForSelectedGradingPeriod(grades: GradeListData) {
```

Comment:
it seems off to name this `get...`
```suggestion
   private func setupGradingPeriodIfNeeded(grades: GradeListData) {
```